src/test/scala/zio/interop/JavaSpec.scala

```diff
@@ -34,18 +34,18 @@ object JavaSpec {
       },
       testM("return an `IO` that produces the value from `Future`") {
         val someValue: UIO[Future[Int]] = UIO.effectTotal(CompletableFuture.completedFuture(42))
-        assertM(ZIO.fromFutureJava(someValue).run, succeeds(equalTo(42)))
+        assertM(ZIO.fromFutureJava(someValue), equalTo(42))
```

Comment:
This is unsolicited but I just remove `.run` and `succeeds` - for success case I think it's natural to assert on value.

Reply:
Makes sense.